variants/STM32F7xx/F745Z(E-G)T_F746Z(E-G)(T-Y)_F750Z8T_F756ZG(T-Y)/variant_ETHERCAT_DUINO.h

```diff
@@ -0,0 +1,203 @@
+ * Copyright (c) 2020-2021, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+#pragma once
+
+/*----------------------------------------------------------------------------
+ *        STM32 pins number
+ *----------------------------------------------------------------------------*/
+#define PA8                     11
+#define PA9                     12
+#define PA15                    13
+#define PB10                    16
+#define PB11                    17
+#define PB14                    2
+#define PC6                     7
+#define PC7                     8
+#define PC8                     9
+#define PC9                     10
+#define PC10                    1
+#define PC11                    0
+#define PC13                    48
+#define PC14                    46
+#define PC15                    47
+#define PD5                     14
+#define PD6                     15
+#define PD12                    3
+#define PD13                    4
+#define PD14                    5
+#define PD15                    6
+#define PE2                     53
+#define PE3                     52
+#define PE4                     51
+#define PE5                     50
+#define PE6                     49
+#define PE7                     19
+#define PE8                     18
+#define PE9                     30
+#define PE10                    31
+#define PE11                    28
+#define PE15                    29
+#define PF3                     45
+#define PF4                     44
+#define PF10                    43
+#define PF11                    36
+#define PF12                    35
+#define PF13                    34
+#define PF14                    21
+#define PF15                    20
+#define PG0                     32
+#define PG1                     33
+#define PG2                     26
+#define PG3                     27
+#define PG4                     24
+#define PG5                     25
+#define PG6                     22
+#define PG7                     23
+#define PG9                     37
+#define PG10                    38
+#define PG11                    39
+#define PG12                    40
+#define PG13                    41
+#define PG14                    42
+#define PH0                     112
+#define PH1                     113
+#define PB3                     54
+#define PB8                     55
+#define PB9                     56
+#define PE12                    57
+#define PE13                    58
+#define PE14                    59
+#define PG15                    60
+#define PF2                     61
```

Comment:
Please reorder for better readability.
Moreover some are higher than   #define NUM_DIGITAL_PINS        62
Ex: 
#define PH0                     112
#define PH1                     113

So they should  be removed.

Reply:
Hopefully, everything is fixed now :)

Reply:
Hope so 😉 
Anyway I've only made a quick review 🤣 I will go deeply soon.